modules/cms/lb.tf

```diff
@@ -124,26 +124,12 @@ resource "aws_security_group" "lb_ingress" {
 
   # For Web traffic to services
   ingress {
-    cidr_blocks = ["0.0.0.0/0"]
-    protocol    = "tcp"
-    from_port   = 80
-    to_port     = 80
-  }
-
-  ingress {
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks =  ["${var.ingress_ips}"]
```

Comment:
Since the `"${var.ingress_ips}"` is a list, this may lead to a list within a list, like so: `[["52.203.194.136/32", "52.20.26.200/32", "34.196.35.156/32"]]`. Running `terraform plan` should tell you if this will work as currently defined. If not, then you can try just dropping the brackets here.

Reply:
I ran this yesterday on a fresh stack (terraform apply straight up) and it didn't error out and produced the security group as anticipated.

Reply:
oh cool. good to know